examples/embed_cxx/README.md

```diff
@@ -0,0 +1,18 @@
+This is a example for embedding WasmEdge Sdk in C++, and how to build WebAssembly form C++ sources.
```

Comment:
This is `an` example `of` embedding WasmEdge `SDK` in C++, and how to build WebAssembly `from` C++ sources.